tests/functional/tree.t

```diff
@@ -49,10 +49,21 @@ Since the following custom arguments are incompatible with the default IQ-TREE a
   $ ${AUGUR} tree \
   >  --method iqtree \
   >  --alignment tree/full_aligned.fasta \
-  >  --tree-builder-args="-czb -bb 1000 -bnni" \
+  >  --tree-builder-args="--polytomy -bb 1000 -bnni --redo" \
   >  --override-default-args \
   >  --output "$TMP/tree_raw.nwk" > /dev/null
 
+Build a tree with conflicting default arguments.
```

Comment:
for the sake of symmetry, might be worth adding one additional test that is identical to this plus `--override-default-args` -- or instead having this test duplicate the arguments in the test above _without_ `--override-default-args`.

Reply:
These cram tests are the slow part of the test suit so I'd probably prefer not to add tests just for symmetry sake - if the tests were fast, then sure!